MAUI/ListView/swiping.md

```diff
@@ -0,0 +1,628 @@
+          <Grid VerticalOptions="Center" HorizontalOptions="Center">
+            <Image Grid.Column="0"
+                   Grid.Row="0"
+                   BackgroundColor="Transparent"
+                   HeightRequest="35"
+                   WidthRequest="35"
+                   BindingContextChanged="leftImage_BindingContextChanged"
+                   Source="Favorites.png" />
+          </Grid>
+        </Grid>
+        <Grid BackgroundColor="#DC595F" HorizontalOptions="Fill" VerticalOptions="Fill" Grid.Column="1">
+          <Grid VerticalOptions="Center" HorizontalOptions="Center">
+            <Image Grid.Column="0"
+                   Grid.Row="0"
+                   HeightRequest="35"
+                   WidthRequest="35"
+                   BackgroundColor="Transparent"
+                   BindingContextChanged="rightImage_BindingContextChanged"
+                   Source="Delete.png" />
+          </Grid>
+        </Grid>
+      </Grid>
+    </DataTemplate>
+  </syncfusion:SfListView.StartSwipeTemplate>
+ </syncfusion:SfListView>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitel="MainPage.xaml.cs" %}
+listView.StartSwipeTemplate = new DataTemplate(() =>
+{
+  var grid = new Grid();
+
+  var grid1 = new Grid()
+  {
+  BackgroundColor = Colors.FromHex("#009EDA"),
+  HorizontalOptions = LayoutOptions.Fill,
+  VerticalOptions = LayoutOptions.Fill
+  };
+  var favoriteGrid = new Grid() { HorizontalOptions = LayoutOptions.Center, VerticalOptions = LayoutOptions.Center };
+  var favoriteImage = new Image() { BackgroundColor = Colors.Transparent, HeightRequest = 35, WidthRequest = 35 };
+  favoriteImage.Source = ImageSource.FromResource("Swiping.Images.Favorites.png");
+  favoriteImage.BindingContextChanged += FavoriteImage_BindingContextChanged;
+  favoriteGrid.Children.Add(favoriteImage);
+  grid1.Children.Add(favoriteGrid);
+
+  var grid2 = new Grid()
+  {
+  BackgroundColor = Colors.FromHex("#DC595F"),
+  HorizontalOptions = LayoutOptions.Fill,
+  VerticalOptions = LayoutOptions.Fill
+  };
+  var deleteGrid = new Grid() { HorizontalOptions = LayoutOptions.Center, VerticalOptions = LayoutOptions.Center };
+  var deleteImage = new Image() { BackgroundColor = Colors.Transparent, HeightRequest = 35, WidthRequest = 35 };
+  deleteImage.Source = ImageSource.FromResource("Swiping.Images.Delete.png");
+  deleteImage.BindingContextChanged += DeleteImage_BindingContextChanged;
+  deleteGrid.Children.Add(deleteImage);
+  grid1.Children.Add(deleteGrid);
+
+  grid.Children.Add(grid1);
+  grid.Children.Add(grid2, 1, 0);
+                
+  return grid;
+});
+{%endhighlight %}
+{% endtabs %}
+
+![Customize Start swipe in listview](Images/swiping/maui-listview-working-with-multiple-views-in-swipetemplate.jpg)
+
+To delete the item when `Delete` image is tapped and set favorites to an item when `Favorites` image is tapped, follow the code example shown.
+
+{% tabs %}
+{% highlight c# %}
```

Comment:
add "TabTitle" in all places in this doc

Reply:
resolved